pkgs/games/fteqw/default.nix

```
@@ -0,0 +1,98 @@
+{ lib
+, stdenv
+, pkgs
+, fetchsvn
+, runCommand
+, pkgName ? "fteqw"
+}:
+let
+  releaseFile = {
+    fteqw = pkgName;
+    fteqw-dedicated = "fteqw-sv";
+    fteqcc = pkgName;
+  };
+in stdenv.mkDerivation rec {
+  pname = pkgName;
+  version = "unstable-2022-08-09";
+  src = fetchsvn {
+    url = "https://svn.code.sf.net/p/fteqw/code/trunk";
+    rev = "6303";
+    sha256 = "sha256-tSTFX59iVUvndPRdREayKpkQ+YCYKCMQe2PXZfnTgPQ=";
+  };
+  makeFlags = [
+    "PKGCONFIG=pkg-config"
+    "-C engine"
+  ];
+  buildFlags = lib.optionals (pkgName == "fteqw") [
+    "m-rel"
+  ] ++ lib.optionals (pkgName == "fteqw-dedicated") [
+    "sv-rel"
+  ] ++ lib.optionals (pkgName == "fteqcc") [
+    "qcc-rel"
+  ];
```

```suggestion
  version = "unstable-2022-08-09";

  src = fetchsvn {
    url = "https://svn.code.sf.net/p/fteqw/code/trunk";
    rev = "6303";
    sha256 = "sha256-tSTFX59iVUvndPRdREayKpkQ+YCYKCMQe2PXZfnTgPQ=";
  };

  makeFlags = [
    "PKGCONFIG=pkg-config"
    "-C engine"
  ];

  buildFlags = lib.optionals (pkgName == "fteqw") [
    "m-rel"
  ] ++ lib.optionals (pkgName == "fteqw-dedicated") [
    "sv-rel"
  ] ++ lib.optionals (pkgName == "fteqcc") [
    "qcc-rel"
  ];

```
some whitespacing can improve readability a lot.